paasta_tools/marathon_tools.py

```diff
@@ -368,6 +370,28 @@ def format_marathon_app_dict(self):
 
         net = get_mesos_network_for_net(self.get_net())
 
+        try:
+            constraints = self.get_calculated_constraints(
+                system_paasta_config=system_paasta_config,
+                service_namespace_config=service_namespace_config
+            )
+        except NoSlavesAvailableError as e:
+            cluster = system_paasta_config.get_cluster()
+            error_msg = (
```

Comment:
I appreciate the sentiment, but alerts are hard to do really well. Specifically this alert will never "resolve".

How about just `_log` instead?